package.json

```diff
@@ -20,53 +20,53 @@
   "license": "MIT",
   "dependencies": {
     "@types/mongodb": "^3.5.27",
-    "bson": "^1.1.4",
+    "bson": "^4.4.0",
     "kareem": "2.3.2",
-    "mongodb": "3.6.8",
+    "mongodb": "3.6.9",
     "mongoose-legacy-pluralize": "1.0.2",
     "mpath": "0.8.3",
     "mquery": "3.2.5",
-    "ms": "2.1.2",
+    "ms": "2.1.3",
     "regexp-clone": "1.0.0",
     "safe-buffer": "5.2.1",
-    "sift": "13.5.2",
+    "sift": "13.5.4",
     "sliced": "1.0.1"
   },
   "devDependencies": {
-    "@babel/core": "7.10.5",
-    "@babel/preset-env": "7.10.4",
-    "@typescript-eslint/eslint-plugin": "4.10.0",
-    "@typescript-eslint/parser": "4.10.0",
+    "@babel/core": "7.14.6",
+    "@babel/preset-env": "7.14.5",
+    "@typescript-eslint/eslint-plugin": "4.27.0",
+    "@typescript-eslint/parser": "4.27.0",
     "acquit": "1.x",
-    "acquit-ignore": "0.1.x",
+    "acquit-ignore": "0.2.0",
     "acquit-require": "0.1.x",
-    "babel-loader": "8.1.0",
+    "babel-loader": "8.2.2",
     "benchmark": "2.1.4",
-    "bluebird": "3.5.5",
-    "chalk": "2.4.2",
-    "cheerio": "1.0.0-rc.2",
+    "bluebird": "3.7.2",
+    "chalk": "4.1.1",
+    "cheerio": "1.0.0-rc.10",
     "co": "4.6.0",
-    "dox": "0.3.1",
-    "eslint": "7.1.0",
-    "eslint-plugin-mocha-no-only": "1.1.0",
-    "highlight.js": "9.18.2",
+    "dox": "0.9.0",
```

Comment:
This will unfortunately break our docs build, so can't upgrade this one currently.